addons/kommander/1.4/kommander.yaml

```
@@ -49,6 +49,10 @@ spec:
     version: 0.38.6
     values: |
       ---
+      federate:
+        kubeaddonsInitializer:
```

```suggestion
        initializer:
```

https://github.com/mesosphere/yakcl/pull/445/files#diff-611e1faa66fced123c06758ba8fe2b40c2889e563937296cfb8e262b1ecc6305R225

---

Once https://github.com/mesosphere/charts/pull/1280 is merged it should open a bump pr, we can just use that pr and I'll cherry-pick this commit over